File: 08 - Pizza Party/Python/main.py
noPeople = int(input("How many people? "))
noPizza = int(input("How many pizzas? "))
noSlices = int(input("How many slices per pizza? "))

print("{} people with {} pizzas, each of {} slices".format(noPeople, noPeople, noSlices))
print("Each person gets {} slices of pizza.".format((noPizza * noSlices)//noPeople))
print("There are {} leftover slices.".format((noPizza * noSlices)%noPeople))

File: 08 - Pizza Party/C++/main.cpp
#include <iostream>

using namespace std;

int main(void)
{
    int people, pizzas, slices = 0;

    cout << "How many people? ";
    cin >> people;
    cout << "How many pizzas? ";
    cin >> pizzas;
    cout << "How many slices per pizza? ";
    cin >> slices;

    cout << endl;

    cout << people << " people with " << pizzas << " pizzas, " << slices << " each." << endl;
    cout << "Each person gets " << ( pizzas * slices ) / people << " pieces of pizza." << endl;
    cout << "There are " << ( pizzas * slices ) % people << " pieces left over." << endl;

    return 0;
}
